llmware/embeddings.py

```diff
@@ -29,7 +29,7 @@
 try:
     from pgvector.psycopg import register_vector
     import psycopg
-except:
+except ImportError:
     pass
```

Comment:
@doberst, the pass is still in place. I've only replaced the bare except with `except ImportError`.

Reply:
Got it.   :).  My bad - should have had my second cup of coffee first ... 😄